daemon/cmd/vitals.go

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"time"
+
+	"github.com/go-openapi/runtime/middleware"
+	"k8s.io/apimachinery/pkg/util/duration"
```

Comment:
I am a bit confused with the heuristics to track dependencies. On previous commit I was ask to specifically keep dependencies update in separate commit. But here since it's small it's ok to roll it in?